amaranth/hdl/_nir.py

```diff
@@ -211,9 +211,9 @@ def __repr__(self):
             result.append(f"(cell {cell_idx} {cell.module_idx} {cell!r})")
         return "\n".join(result)
 
-    def add_module(self, parent, name: str):
+    def add_module(self, parent, name: str, *, src_loc=None, cell_src_loc=None):
```

Comment:
I guess this works because there's a 1:1 relationship between modules and instantiations, but it's a little weird. (I don't think there is a strong need to change this.)

Reply:
Well we don't really have any other place to put it, so...